extras/rya.manual/src/site/markdown/rya-streams.md

```diff
@@ -0,0 +1,385 @@
+2. Rya Streams notices the change starts/stops processing a query based on
+   what the change was.
+3. Statements are discovered for ingest by whatever application is loading data
+   into Rya.
+4. Those Statements are written to Rya Streams so that the Streams Queries may
+   process them and produce results.
+5. Those Statements are also written to the Rya instance for ad-hoc querying.
+6. Rya Streams produces Visibility Binding Sets and/or Visibility Statements 
+   that are written back to Rya.
+7. Those same Visibility Binding Sets and/or Visibility Statements are made
+   available to other systems as well. 
+
+<div id='quick-start'/>
+
+## Quick Start ##
+This tutorial demonstrates how to install and start the Rya Streams system. It
+must be configured and running on its own before any Rya instances may use it. 
+After performing the steps of this quick start, you will have installed the 
+system, demonstrated that it is functioning properly, and then may use the Rya 
+Shell to configure Rya instances to use it.  
+ 
+This tutorial assumes you are starting fresh and have no existing Kafka, or 
+Zookeeper data. The two services must already be installed and running.
+
+### Step 1: Download the applications ###
+
+You can fetch the artifacts you need to follow this Quick Start from our
+[downloads page](http://rya.apache.org/download/). Click on the release of
+interest and follow the "Central repository for Maven and other dependency 
+managers" URL.
+
+Fetch the following two artifacts:
+
+Artifact Id | Type 
+--- | ---
+rya.streams.client | shaded jar
+rya.streams.query-manager | rpm
+
+### Step 2: Install the Query Manager ###
+
+Copy the RPM to the CentOS 7 machine the Query Manager will be installed on.
+Install it using the following command:
+
+```
+yum install -y rya.streams.query-manager-3.2.12-incubating.noarch.rpm
+```
+
+It will install the program to **/opt/rya-streams-query-manager-3.2.12**. Follow
+the directions that are in the README.txt file within that directory to finish
+configuration. 
+
+### Step 3: Register a Streams Query with Rya Streams ###
+
+Use the Rya Streams Client to register the following Streams Query:
+
+```
+SELECT * 
+WHERE { 
+    ?person <urn:talksTo> ?employee .
+    ?employee <urn:worksAt> ?business 
+}
+```
+We assume Kafka is running on the local machine using the standard Kafka port.
+Issue the following command:
+
+```
+java -jar rya.streams.client-3.2.12-incubating-SNAPSHOT-shaded.jar add-query \
+    -i localhost -p 9092 -r rya-streams-quick-start -a true \
+    -q "SELECT * WHERE { ?person <urn:talksTo> ?employee .?employee <urn:worksAt> ?business }"
+```
+The Query Manager should eventually see that this query was registered and 
+start a Rya Streams job that will begin processing it using any Visibility 
+Statements that have been loaded into Rya Streams. If no results are observed
+in step 6 , then verify the Query Manager is working properly. It's logs can be
```

Comment:
Change to `[Step 6](#step-6:-observe-the-results-that-appear)` and remove space before comma.

Also, typo. "Its log"

Reply:
Done, but not linking.